atd-vzd/triggers/fatality_insert.sql

```diff
@@ -0,0 +1,26 @@
+-- This function inserts a new record into the fatality table
+-- It is triggered by fatal insertions to the primaryperson or person tables
+CREATE OR REPLACE FUNCTION fatality_insert() 
+    RETURNS trigger
+    LANGUAGE plpgsql
+AS $function$
+    DECLARE
+    	year integer;
+        location text;
+        crash_position geometry;
+    BEGIN
+        SELECT TO_CHAR(crash_date,'yyyy'), CONCAT_WS(' ', rpt_block_num, rpt_street_pfx, rpt_street_name, '(', rpt_sec_block_num, rpt_sec_street_pfx, rpt_sec_street_name, ')'), position
```

Comment:
if y'all use this same concatenation elsewhere, it might be best to add it as a generated column on the `atd_txdot_crashes` table.

Reply:
TIL generated columns